modules/zenith-public/lua/1-enum/zxi/mobHelpers.lua

```diff
@@ -22,4 +22,51 @@ zxi.mobHelpers.getRespawnVarName = function(mob, mobId)
     return fmt(template, mob:getName(), mob:getID())
 end
 
+zxi.mobHelpers.zoneInitReposition = function(zone, mobList)
+    if not zone or not mobList then
+        return
+    end
```

Comment:
while we're doing a guard clause, may as well ensure we don't shoot ourselves in the foot

```suggestion
    if not zone or type(mobList) ~= 'table' then
        print('zxi.mobHelpers.zoneInitReposition : invalid parameters specified')
        print(zone)
        print(mobList)

        return
    end
```